### PR TITLE
Make v0.14.8

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Ports the work done in #4259 to `v0.14.x`, so we can make `v0.14.8`.

---

**Checklist:**

- [x] Added a file to CHANGELOG.d for this PR (see CHANGELOG.d/README.md)
- [ ] Added myself to CONTRIBUTORS.md (if this is my first contribution)
- [x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: "CI"
 
 on:
   push:
-    branches: [ "master" ]
+    branches: [ "master", "continue-v0.14.x" ]
   pull_request:
-    branches: [ "master" ]
+    branches: [ "master", "continue-v0.14.x" ]
```
Comment (thomashoneyman) [inline .github/workflows/ci.yml:7]: Is this necessary?

Comment (JordanMartinez) [reply .github/workflows/ci.yml:7]: CI wasn't building in this PR and I wanted to be sure that `libtinfo` was verified by CI to not be included in the binary. So, I think so? I can remove it in a later commit once CI builds.

Comment (rhendric) [reply .github/workflows/ci.yml:7]: I assume `continue-v0.14.x` will only receive backports and isn't going to be merged back into `master`, so it doesn't matter if it's ever removed, right? This is meant to be a short-lived branch, right?

Comment (JordanMartinez) [reply .github/workflows/ci.yml:7]: Yup! I don't anticipate making a `v0.14.9` release. This is just to fix the `libtinfo` regression in `v0.14.7`

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**